velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -197,4 +197,34 @@ bool HdfsFileSystem::exists(std::string_view path) {
   return impl_->hdfsShim()->Exists(impl_->hdfsClient(), path.data()) == 0;
 }
 
+void HdfsFileSystem::rename(
+    std::string_view path,
+    std::string_view newPath,
+    bool overWrite) {
```

Comment:
Is the `overWrite` argument supported?

Reply:
@majetideepak No. Hdfs doesn't support `overwrite`.https://github.com/facebookincubator/velox/blob/dbe4d46a05982082bed1be2100d4ee566865bd31/velox/external/hdfs/ArrowHdfsInternal.h#L92.